GentWebApi/Controllers/UsersController.cs

```diff
@@ -22,15 +23,19 @@ public class UsersController : Controller {
 			User response = _context.Users
 				.Include(u => u.Company)
 				.ThenInclude(c => c.Branches)
+				.ThenInclude(b => b.Events)
+				.Include(u => u.Company)
```

Comment:
gaf include company geen problemen?

Reply:
Nee hier halen we gewoon het volledige object op. Moest je onder branches nog eens theninclude company zetten dan zou hij wel problemen hebben met die circular dependency